Makefile

```diff
@@ -1,8 +1,8 @@
 .PHONY: bootstrap-init mgmt-deploy mgmt-destroy build-api-image push-api-image deploy-tre destroy-tre letsencrypt
 
 SHELL:=/bin/bash
-ROOTPATH:=$(shell pwd)
-
+MAKEFILE_FULLPATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+CURRENT_DIR := $(dir $(MAKEFILE_FULLPATH))
```

Comment:
nit: `CURRENT_DIR`  _could_ be interpreted as the current working directory, so perhaps `MAKEFILE_DIR` or something similar might more clearly express "the directory that this Makefile is in"?